src/main/java/com/srans/nestserver/controller/NotificationController.java

```diff
@@ -27,23 +30,30 @@
 	@Autowired
 	private NotificationRepository notificationRepo;
 	
+	@Autowired
+	private VacationRepository vacationRepo;
+	
 	
  
 	//Get Notification	
-	@GetMapping(value = "/users/{userId}/notifications")
+	@GetMapping(value = "/users/vacate")
 	@PreAuthorize("permitAll()")
-	public List<Notification> getAllUserNotification(@PathVariable(value = "userId") Long userId) throws NSException {
+	public List<VacationInfo> getAllUserNotification(@RequestParam Long adminId) throws NSException {
 		logger.info("In::getAllNotificationsOfUser::" );
-		List<Object[]> response = notificationRepo.getAllNotification(userId);
+		List<Object[]> response = notificationRepo.getAllNotification(adminId);
 		System.out.println(response.size());
-		List<Notification> l1 =new ArrayList<>();
-		Notification notifications = null;
+		List<VacationInfo> l1 =new ArrayList<>();
```

Comment:
Variable name not looks good, plz use correct names

Reply:
I have renamed reference variable